astropy/time/tests/test_precision.py

```diff
@@ -513,7 +513,7 @@ def test_jd_add_subtract_round_trip(scale, jds, delta):
 
 @given(scale=sampled_from(STANDARD_TIME_SCALES),
        jds=reasonable_jd(),
-       delta=floats(-3 * tiny, 3 * tiny).filter(lambda x: x == 0 or abs(x) > (3 * tiny)))
+       delta=floats(-3 * tiny, 3 * tiny).filter(lambda x: x == 0 or (0.5 > abs(x) > (3 * tiny))))
```

Comment:
I think this filter only allows x == +/- zero, because the value is upper bounded by 3*tiny so the second clause is always false.

Reply:
Duh, you are right. I think at this point I am just very confused and not sure what this test is trying to do with the delta values...